calcom/cal.com
# Set GCal & Outlook `iCalUID` in .ics file

## What does this PR do?



This PR fixes Gmail parsing our .ics file and adding double events in GCal by adding the iCalUID when creating events to the .ics file we generate.

Fixes # (issue)




**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



- Connect GCal
- Create an event
- From the email get the .ics file
- Send the .ics to your inbox. When clicking "View on Google Calendar" it should open the event that we created
- Reschedule that booking
- The new .ics file should still point to the event that we created

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
